pkg/cluster/arooperator.go

```diff
@@ -36,16 +36,15 @@ func (m *manager) aroDeploymentReady(ctx context.Context) (bool, error) {
 	return m.aroOperatorDeployer.IsReady(ctx)
 }
 
-func (m *manager) ensureAROOperatorRunningDesiredVersion(ctx context.Context) error {
+func (m *manager) ensureAROOperatorRunningDesiredVersion(ctx context.Context) (bool, error) {
 	if !m.isIngressProfileAvailable() {
```

Comment:
Why are we checking the ingress profile is available?

Reply:
Nevermind - I see that we don't deploy/update the ARO operator if the ingress profile is not available, so all these checks would fail anyway. 

Can you put a comment above this stating that?